pkg/controller/clusterlogging/clusterlogging_controller.go

```diff
@@ -82,9 +82,9 @@ func (r *ReconcileClusterLogging) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, err
 	}
 
-	if instance.Spec.ManagementState == loggingv1.ManagementStateUnmanaged {
-		return reconcile.Result{}, nil
-	}
+	//if instance.Spec.ManagementState == loggingv1.ManagementStateUnmanaged {
```

Comment:
I would expect a check like this and something similar may also be needed in the other controllers.  Essentially:
* Managed or blank or nil = "Managed" - continue
* Unmanaged = "Unmanaged" - do nothing and return early

Reply:
In case we do: `Unmanaged = "Unmanaged" - do nothing and return early`.
[k8shandler.Reconcile()](https://github.com/openshift/cluster-logging-operator/blob/master/pkg/controller/clusterlogging/clusterlogging_controller.go#L89) will never be called even for create (this is how it works now in master) and instances will not be created

Reply:
yes.  if you are initially creating an "unmanaged" clusterlogging instance it is reasonable we do nothing; that is what you told us